tests-beta/android/MASVS-PLATFORM/MASTG-TEST-0x24.md

```diff
@@ -0,0 +1,33 @@
+---
+platform: android
+title: Testing for App Permissions
+id: MASTG-TEST-0x24
+---
+
+## Overview
+
+Testing for app permissions in Android involves evaluating how an application requests, uses, and manages permissions to ensure they do not lead to security vulnerabilities. Proper permission management should protect sensitive user data and ensure that the application complies with Android's security model. The test aims to detect misconfigurations and unnecessary permissions.
+
+## Steps
+
+### Static analysis
```

Comment:
We do not use these sections anymore but rather one unique list of steps.

In this case you can easily create a MASTG-TECH for "Obtaining App Permissions" and there use one ## section per tool.

Example:

`## @MASTG-TOOL-xxx`

See the latest MASTG-TECH and MASTG-TEST (V2 beta) for reference (you can go to the page and sort the IDs to see the recent ones).

FOR THIS TEST:

as you'll see in the other ones this will become one step about using the technique by referencing it with @